src/app/profile/meditated-recently.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+import * as moment from 'moment';
+
+@Directive({
+  selector: '[meditated-recently]'
+})
+export class MeditatedRecentlyDirective {
+
+  _meditatedRecently: boolean = false;
+
+  @Input() user;
+
+  @HostBinding('class.meditated-recently')
+  get meditatedRecently() { return this._meditatedRecently; }
+
+  ngOnChanges() {
+    if (!this.user || !this.user.lastMeditation) {
+      return;
+    }
+
+    const diff = moment().diff(moment(this.user.lastMeditation));
+    const duration = moment.duration(diff).asHours();
+    this._meditatedRecently = duration < 2;
```

Comment:
Just thought it was pretty long having the borders for three hours, but it makes sense to align it with the meditation tab. Will change to 3 hours.